packages/cozy-harvest-lib/src/components/OAuthForm.jsx

```diff
@@ -72,30 +72,31 @@ export class OAuthForm extends PureComponent {
   }
 
   render() {
-    const { konnector, t, flowState } = this.props
-    const { initialValues, showOAuthWindow, needExtraParams, extraParams } =
-      this.state
+    const { konnector, t, flowState, reconnect, account } = this.props
+    const { showOAuthWindow, needExtraParams, extraParams } = this.state
     const isBusy =
       showOAuthWindow === true ||
       flowState.running ||
       (needExtraParams && !extraParams)
 
-    return initialValues ? null : (
```

Comment:
Why are you removing this check? Does the `initialValues` still needed at all in this component?

Reply:
InitialValues was only needed to hide the "connect" button when trying to reconnect an oauth account. Now, we have the reconnect property and we can display the button in any case.

Reply:
I removed other references to initialValues.